src/autocomplete.js

```diff
@@ -126,6 +126,7 @@ export default class Autocomplete {
     const value = selected.getAttribute('data-autocomplete-value') || selected.textContent
     this.container.value = value
     this.container.open = false
+    this.container.dispatchEvent(new CustomEvent('commit', {detail: {value}}))
```

Comment:
This should already be available with the `change` event, right?

Reply:
OMG, I justr searched for `dispatchEvent` in `src/autocomplete.js` and not `src/autocomplete-element.js` 🤦‍♂️ I'll see if that event works with my `github/github` branch and close this PR if that's the case.

Reply:
Works perfectly, thanks @dgraham 💖